.github/CHANGELOG.md

```diff
@@ -246,6 +246,8 @@
 1. [FMGC] Implemented Airport Button - @patmack14 (Patrick Macken)
 1. [RMP] Fixed colour of SEL indicator - @tracernz (Mike)
 1. [MISC] Added custom Autobrake event for SimConnect - @aguther (Andreas Guther)
+1. [EWD] Fixed failures not re-triggering when resolved - @tricky_dicky (Richard Pilbery)
```

Comment:
Shouldn't be here I guess? :)

Reply:
Well, it got missed off the PR it relates too and has been merged, so the changelog needs updating...seemed a bit overkill to make it a separate PR.

Reply:
Fair point^^